server/app/views/admin/AdminLayout.java

```diff
@@ -89,6 +93,7 @@ private Content render(HtmlBundle bundle, boolean isCentered) {
     bundle.addMainStyles(
         AdminStyles.MAIN, isCentered ? AdminStyles.MAIN_CENTERED : AdminStyles.MAIN_FULL);
     bundle.addBodyStyles(AdminStyles.BODY);
+    maybeAddSessionTimeoutModals(bundle, messagesApi.preferred(bundle.getRequest()));
```

Comment:
```suggestion
    addSessionTimeoutModals(bundle, messagesApi.preferred(bundle.getRequest()));
```

(blocking) From above